### DISCUSSION TITLE
How to use splice/unsplice proportion and phase plot to judge the applicability of scVelo to a dataset

### DISCUSSION THREAD

**Discussion by gundalav** [theislab/scvelo#573] [Q&A]
Hi, 

I have the following data set that gives this proportion:

<img width="606" alt="Screen Shot 2021-07-30 at 14 00 55" src="https://user-images.githubusercontent.com/8143832/127603263-56224401-65f6-47c4-82de-8c765a98648d.png">

The phase plot looks like this:

<img width="1265" alt="Screen Shot 2021-07-30 at 13 58 53" src="https://user-images.githubusercontent.com/8143832/127603309-7aa38202-c4d3-4036-a42c-d2c7694c38a6.png">

Currently the final velocity embedding stream looks like this:

<img width="419" alt="Screen Shot 2021-07-30 at 13 57 35" src="https://user-images.githubusercontent.com/8143832/127603405-15a3e39f-0cda-4d25-a8c9-2401bac6b588.png">

My question is based on the near 50/50 splice/unspliced proportion and phase plot.
Can we deem whether or not scVelo can be used in our dataset?

**Comment by WeilerP** [comment]
There is no clear football shape (or part thereof) in your phase portraits, i.e. the fundamental assumption of parameter inference with scVelo is violated. This has already been extensively discussed in numerous other issues and discussions. See e.g. #216, #462, #472.

**Reply by gundalav** [reply]
@WeilerP Thanks for your reply. Can you give example of  the ideal "football/almond" shape?

**Reply by WeilerP** [reply]
You can see them in the paper or it's supplement, for example.
